Makefile

```diff
@@ -279,7 +279,7 @@ build-llvm-profiling: KOMPILE_OPTS += -O3 -ccopt -fno-omit-frame-pointer
 build-llvm-profiling: $(KPLUTUS_LIB)/$(llvm_kompiled)
 
 bison_version    := $(shell eval "bison --version | head -n1 | sed 's/bison (GNU Bison) //g'")
-smallest_version := $(shell echo "$(bison_version)\n3.7.4" | sort -V | head -n1)
+smallest_version := $(shell echo -e "$(bison_version)\n3.7.4" | sort -V | head -n1)
```

Comment:
There seems to be some weird behavior here with make where echo prints the `-e` instead of taking it as a parameter. I think we can just use `printf` here.